app/models/video.rb

```diff
@@ -11,4 +15,14 @@ def self.search_by_title(search_term)
     self.where("title LIKE ?", "%#{search_term}%")
     #find(:all, :conditions => ['title LIKE ?', "%#{search_term}%"]) This is not working
   end
+
+  def video_url_string
```

Comment:
This method feels more like a presentation logic and not business logic. Maybe a helper in views would be better or Decorator. About decorator - you will learn about them in lesson 7 2nd part.